docs/index.md

```diff
@@ -29,4 +29,31 @@ See the relationship between the connector files (in blue) and the Tableau **Con
 You need to do these things before you start:
 
 - Install the ODBC or JDBC driver that you’ll use with the connector you’ll create.
-- Install Tableau Desktop 2019.2 or later on a Windows or Mac computer.
\ No newline at end of file
+- Install Tableau Desktop 2019.2 or later on a Windows or Mac computer.
+
+## Using a Connector
+You must start Tableau Desktop or Server with a special command line argument which tells Tableau where to find your Connector. See [Share Your Connector]({{ site.baseurl }}/docs/share) for more information.
+
+**Tableau Desktop:** 
+
+For Windows:
+1. Create a directory for Tableau connectors. For example: `C:\tableau_connectors`
+1. Put the folder containing your connector's manifest.xml file in this directory. Each connector should have its own folder. For example: `C:\tableau_connectors\my_connector`
+1. Run Tableau using the `-DConnectPluginsPath` command line argument, pointing to your connector directory. For example: 
+
+    ```
+    tableau.exe -DConnectPluginsPath=C:\tableau_connectors
+    ```
+
+For macOS:
+
+Replace [user name] with your name, for example /Users/kgarcia/tableau_connectors, and [Tableau version] with the version of Tableau that you’re running, for example, 2019.1.app.
```

Comment:
Replace with:
"In the following examples, replace [user name] with your name (for example, /Users/agarcia/tableau_connectors) and [Tableau version] with the version of Tableau that you're running (for example, 2019.3.app)."
I propose these changes because "agarcia" fits in our guidelines for fictitious information, and I thought we might want a Tableau version that's later than the minimum listed above.